pypi_count/py_counter.py

```diff
@@ -83,3 +90,18 @@ def count_classes_with_docstring(self):  # pylint: disable=R1710
                 total.append(count)
                 count += 1
         return count
+
+    def count_function_parameters(self):
+        """Counting the parameters within a function."""
+        functions = m.findall(self.search, m.FunctionDef())
+        param_result = 0
+        for func in functions:
+            if func.name.value == self:
```

Comment:
This should check if `func.name.value` equals the name of the function given to this function as an argument. Once a parameter is added to take in the name of the function to look for, this `self` should be updated to be the name of that parameter.

Reply:
@caldwella2